site/src/pages/components/combobox.explainer.mdx

```diff
@@ -142,80 +110,15 @@ To showcase the selected option within a `combobox`, you can include a `<selecte
             <option>Strawberry</option>
             <option>Blueberry</option>
         </optgroup>
-    </listbox>
+    </datalist>
 </combobox>
 ```
 
-## Introducing `filter` attribute
```

Comment:
Why did you remove this? Let's include the new attributes and then determine how they want them to be incubated?

Reply:
@gregwhitworth Added back.